website/Gemfile

```diff
@@ -1,3 +1,3 @@
 source "https://rubygems.org"
 
-gem "middleman-hashicorp", "0.3.4"
+gem "middleman-hashicorp", "0.3.11"
```

Comment:
@jasoncostello can you use 0.3.12 here please? Also, please update `packer.json` to use 0.3.12.